### PR TITLE
22: cassandra-quarkus fixes once we get a driver with JAVA-2663

### PR DESCRIPTION
**Pull request by absurdfarce**

### CODE REVIEW

**Comment by absurdfarce** [comment]
Manual testing confirmed that with these changes + the java-driver changes in https://github.com/datastax/java-driver/pull/1452 the quickstart app functions as expected.  Runtime init of os.Native is a very nice win here (and something I should've fixed long ago).

**Comment by absurdfarce** [comment]
Build failures are caused by the use of a 4.7.0-SNAPSHOT which doesn't include the changes in JAVA-2803.  Once we get that merged and publish a new artifact this build should be fine.

**Comment by absurdfarce** [comment]
JAVA-2803 has now been merged to java-driver so we _should_ have a clean build here once a new snapshot artifact is published.

File: quickstart/setup_db.cql [hunk 1]
```diff
@@ -1,2 +1,2 @@
 CREATE KEYSPACE IF NOT EXISTS k1 WITH replication = {'class':'SimpleStrategy', 'replication_factor':1};
-CREATE TABLE IF NOT EXISTS k1.fruit(store_id, name text, description text, PRIMARY KEY((store_id), name));
\ No newline at end of file
+CREATE TABLE IF NOT EXISTS k1.fruit(store_id text, name text, description text, PRIMARY KEY((store_id), name));
```
Comment (tomekl007) [inline quickstart/setup_db.cql:2]: This script is removed on the current `master` so please revert this change, otherwise, you'll get a confict

**Comment by tomekl007** [comment]
I've resolved conflicts with master
